frontend/prebuild/src/app/login/login.component.ts

```diff
@@ -146,13 +165,38 @@ export class LoginComponent implements OnInit {
       return;
     }
     this.player.name = username;
+    this.username = username;
     sessionStorage.setItem('username', username);
     this.pane = 'right';
   }
+  async loginThroughGoogle() {
+    let jwt = localStorage.getItem("jwt");
+    let user: any = await this.http.get(`${environment.API_URL_PLAYERS}/getJWTInfo/${jwt}`, { responseType: 'json', headers: new HttpHeaders({
+        'Content-Type':  'application/json',
+        'Authorization': 'Bearer ' + `${jwt}`
+    }) }).toPromise(); 
+    if (user.exists === 'true') {
+      sessionStorage.setItem('username', user.username);
+      sessionStorage.setItem('userId', user.id);
+	  this.player.name = user.username;
+    } else {
+      var username = prompt("Choose a username:", "");
```

Comment:
Ideally we would show the middle pane and maybe have a caption like "choose a user name" and have the button say "register" or something instead of the modal. If you don't want to go through all that mess I can probably take care of it later.

Reply:
Let's leave this as a todo for now that one of us can pick up later.